provider/scripts/k8s/offline_store_pandas_runner.py

```diff
@@ -66,21 +66,22 @@ def execute_sql_job(mode, output_uri, transformation, source_list, blob_credenti
                 globals()[f"source_{i}"]= pd.read_csv(output_path)
             else:
                 globals()[f"source_{i}"]= pd.read_parquet(output_path)
+        
         mysql = lambda q: sqldf(q, globals())
         output_dataframe = mysql(transformation)
         print(output_dataframe.head())
         dt = datetime.now()
-        output_uri_with_timestamp = f'{output_uri}{dt}'
+        output_uri_with_timestamp = f'{output_uri}{dt}.parquet'

```

Comment:
should actually be a trailing '/' and one before {dt} I think so the datetime itself is a directory